source/features/deprioritize-marketplace-link.tsx

```diff
@@ -1,30 +1,33 @@
 import React from 'dom-chef';
 import select from 'select-dom';
-import onetime from 'onetime';
-import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
-import onProfileDropdownLoad from '../github-events/on-profile-dropdown-load';
 
 async function init(): Promise<void> {
-	const marketplaceLink = await elementReady('.Header-link[href="/marketplace"]', {waitForChildren: false});
-	if (marketplaceLink) { // On GHE it can be disabled #3725
-		// The link seems to have an additional wrapper that other links don't have https://i.imgur.com/KV9rtSq.png
-		marketplaceLink.closest('.border-top, .mr-3')!.remove();
+	const marketplaceLink = select('.Header-link[href="/marketplace"]');
+	// On GHE it can be disabled #3725
+	if (!marketplaceLink) {
+		return;
+	}
+
+	marketplaceLink.remove();
 
-		await onProfileDropdownLoad();
-		select.last('.header-nav-current-user ~ .dropdown-divider')!.before(
-			<div className="dropdown-divider"/>,
-			<a role="menuitem" className="dropdown-item" href="/marketplace">Marketplace</a>,
-		);
+	const link = <a href="/marketplace">Marketplace</a>;
+	const item = <li>{link}</li>;
+
+	if (pageDetect.isDashboard()) {
+		link.classList.add('Link--secondary');
+	} else {
+		item.classList.add('ml-3', 'ml-lg-0');
 	}
+
+	select.last('.footer ul')!.append(item);
 }
 
 void features.add(import.meta.url, {
 	exclude: [
 		pageDetect.isGist,
 	],
-	awaitDomReady: false,
-	init: onetime(init),
+	init,
```

Comment:
This is right because the footer needs to be updated every time, but the header remains unchanged across ajaxed loads, so after the first run, `init` will just return on line 11 I think

Reply:
Do you mean `onetime` should be kept?